strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenGroupRepositoryProviderTest.java

```diff
@@ -53,11 +43,19 @@
 @SpringBootTest
 @ActiveProfiles(profiles = "test")
 @ContextConfiguration(classes = Maven2LayoutProviderTestConfig.class)
-@Execution(CONCURRENT)
+@Execution(SAME_THREAD)
```

Comment:
It stopped working concurrently ?

Reply:
Yes, but only in Windows... Look at this [Jenkins Job](https://jenkins.carlspring.org/blue/organizations/jenkins/strongbox%2Fbuilds%2Fstrongbox/detail/PR-1370/12/pipeline). There were resource lock issues.

Reply:
Could you try enabling it one more time and let us know if it fails on Jenkins ?

Reply:
Hah! Now it's working! Magic?

Reply:
I think it was Jenkins fault :) 👍